main.js

```diff
@@ -0,0 +1,225 @@
+// 1
+
+function isAlarmSet(isEmployed, isOnVacation) {
+    return isEmployed && !isOnVacation;
+}
+
+// 2
+
+function isOneOrBothTruthy(valueOne, valueTwo) {
+    if (valueOne) {
+        return true;
+    }
+    if (valueTwo) {
+        return true;
+    }
+    return false;
+}
+
+function isOneTruthy(valueOne, valueTwo) {
+    if (valueOne && valueTwo) {
+        return false;
+    }
+    if (valueOne) {
+        return true;
+    }
+    if (valueTwo) {
+        return true;
+    }
+    return false;
+}
+
+// 3
+
+function compareDistanceToMargin(objectA, objectB, margin = 0) {
+    if (Math.abs(objectA - objectB) <= margin) {
+        return 0;
+    }
+    if (objectA > objectB) {
+        return 1;
+    }
+    return -1;
+}
+
+// 4
+
+function isTruthyOrFalsy(trueOrFalse, functionIfTruthy, functionIfFalsy) {
+    if (trueOrFalse) {
+        return functionIfTruthy();
+    }
+    return functionIfFalsy();
+}
+
+// 5
+
+function rentalCarCost(rentalDays) {
+    const costWithoutDiscount = rentalDays * 40;
+
+    if (rentalDays >= 7) {
+        return costWithoutDiscount - 50;
+    }
+    if (rentalDays >= 3) {
+        return costWithoutDiscount - 20;
+    }
+    return costWithoutDiscount;
+}
+
+// 6
+
+function convertAmericanToEuropeanFloor(americanFloor) {
+    if (americanFloor > 12) {
+        return americanFloor - 2;
+    }
+    if (americanFloor > 0) {
+        return americanFloor - 1;
+    }
+    return americanFloor;
+}
+
+// 7
+
+function rockPaperScissorsLizardSpock(playerOne, playerTwo) {
+    function playWithScissors(opponent) {
```

Comment:
There is no need to declare those functions inside of a function - declare them outside. If you feel like it takes a lot of space, you can create a separate file just for the `rockPaperScissorsLizardSpock` function and the functions it uses.